tests/acceptance/percysnapshot.js

```diff
@@ -0,0 +1,63 @@
+import { setupServer, shutdownServer } from './server';
```

Comment:
Is this file a copy of the existing acceptancesuite? Was the reason for that so people don't generate snapshots if they run the tests locally? I figured we'd have one set of tests and we could pass a flag or something to generate snapshots (or not).

Reply:
Snapshots aren't generated unless you run testcafe with a percy --exec. The issue was that the acceptance tests are run 4 times total, once per browser we care about. On the other hand, Percy only wants you to take snapshots once per build, so the percy_snapshots circleci job had to be separate from the regular acceptance_tests job. At that point it felt a little weird to run the entire acceptance suite again, when all that step cared about was taking snapshots, but I can update this PR to put the snapshots into acceptancesuite.js again.

Reply:
Actually I like being able to put snapshots in the main tests more, even if it means running them an extra time. The browserstack automate video is pretty garbage so this can also give some insight into failing tests

Reply:
*Sorry I tried again and running acceptance tests that contain percySnapshot() calls even without `percy exec`ing them is fine EXCEPT in browserstack ie11, so it might be necessary to have the snapshots be separate